server/DAL/Configuration/RoleConfiguration.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace DAL.Configuration;
 
-public class RoleConfiguration : IEntityTypeConfiguration<IdentityRole<int>>
+internal class RoleConfiguration : IEntityTypeConfiguration<IdentityRole<int>>
```

Comment:
why not public?

Reply:
It is only used inside DbContext class. It should not be used anywhere else and AppDbContext will stay in DAL. Why make it public than?